pallets/xcmp-queue/src/benchmarking.rs

```diff
@@ -21,24 +21,17 @@ use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
 use frame_system::RawOrigin;
 
 benchmarks! {
-	// This will measure the execution time of `set_dummy` for b in [1..1000] range.
-	set_config_with_u32 {}: update_resume_threshold(RawOrigin::Root, 100)
+	set_config_with_resume_threshold {}: update_resume_threshold(RawOrigin::Root, 100)
 
-	// This will measure the execution time of `accumulate_dummy` for b in [1..1000] range.
-	// The benchmark execution phase is shorthanded. When the name of the benchmark case is the same
-	// as the extrinsic call. `_(...)` is used to represent the extrinsic name.
-	// The benchmark verification phase is omitted.
 	set_config_with_weights {}: update_xcmp_max_individual_weight(RawOrigin::Root, 3_000_000)
 
+	set_config_with_restrict_decay {}: update_weight_restrict_decay(RawOrigin::Root, 3_000_000)
 
-	// This line generates test cases for benchmarking, and could be run by:
-	//   `cargo test -p pallet-example-basic --all-features`, you will see one line per case:
-	//   `test benchmarking::bench_sort_vector ... ok`
-	//   `test benchmarking::bench_accumulate_dummy ... ok`
-	//   `test benchmarking::bench_set_dummy_benchmark ... ok` in the result.
-	//
-	// The line generates three steps per benchmark, with repeat=1 and the three steps are
-	//   [low, mid, high] of the range.
+	set_config_with_threshold_weight {}: update_threshold_weight(RawOrigin::Root, 3_000_000)
+
+	set_config_with_drop_threshold {}: update_drop_threshold(RawOrigin::Root, 3_000_000)
+
+	set_config_with_suspend_threshold {}: update_suspend_threshold(RawOrigin::Root, 3_000_000)
```

Comment:
We can actually combine all of these benchmarks in a single one, since there isn't any behavioral or weight difference between setting all of these `u32` parameters.

Reply:
Agreed - that's much nicer. I see now how that can be achieved.